chaosaws/rds/actions.py

```diff
@@ -191,6 +191,34 @@ def delete_db_cluster_endpoint(db_cluster_identifier: str,
             db_cluster_identifier, e.response['Error']['Message']))
 
 
+def start_db_instance(db_instance_identifier: str,
+                      configuration: Configuration = None,
+                      secrets: Secrets = None) -> AWSResponse:
+    """
+    Start a RDS DB instance
+
+    - db_instance_identifier: the identifier of the RDS instance to start
+    """
+    client = aws_client("rds", configuration, secrets)
+
+    params = dict(
+        DBInstanceIdentifier=db_instance_identifier)
+
+    try:
+        call = client.start_db_instance(**params)
+
+        while True:
+            instance_status = client.describe_db_instances( \
```

Comment:
should we have two things:

* a timeout just in case it blocks forever?
* a pause between each iteration?

Reply:
I've added a timeout of 600 seconds, and a pause of 5 seconds between each iteration ;)